### ISSUE TITLE
vertical spacing variables in theme are broken

### ISSUE THREAD

**Issue by jsms90** [elifesciences/elife-xpub#236]
**Describe the bug**
Vertical spacing variables have been added to the theme in PR #233 but the values do not evaluate to `px` values - see [@tamlyn's comment on the PR](https://github.com/elifesciences/elife-xpub/pull/233#discussion_r197084939)

Caused by [this code](https://github.com/elifesciences/elife-xpub/blob/develop/client/elife-theme/src/index.js#L44-L50):
```
verticalXxl: 'gridUnit'*20,
verticalXl: 'gridUnit'*12,
verticalL: 'gridUnit'*8,
verticalM: 'gridUnit'*6,
verticalS: 'gridUnit'*4,
verticalXs: 'gridUnit'*2,
verticalXxs: 'gridUnit',
```

So we can't use these variables at the moment.

It might not be the best idea to expose 2 identical variables `verticalXxs` & `gridUnit` anyway?

#### Relevant tickets
- [design issue in pubsweet](https://gitlab.coko.foundation/pubsweet/design/issues/31)
- [original (closed) issue for updating theme] - #183 

#### Suggested solutions
1. How about using `gridUnit` as a measurement (`const` at the top of the file, but not exposed) & only exposing `vertical...`?  
This means that our variables are diverging from Coko's default theme though, in removing `gridUnit`.
2. Remove `verticalXxs`, `verticalXs`, etc altogether & use `gridUnit * 2` (for example), in the same way that other projects already do

---

Another potential concern is that we're using these "vertical spacing" variables for padding/margin left and right as well. So they're not just "vertical". Perhaps `gridUnit` is a better name?

We could have `gridUnitXxl` ... `gridUnitXxs`?
Note: this still means getting rid of `gridUnit` and replacing it with `gridUnitXxs`

**Comment by tamlyn** [comment]
`gridUnit` is a core PubSweet theme variable and cannot be removed without breaking the PubSweet components that we use. Also, there are some values which should probably be derived from the `gridUnit` but are no t-shirt sizes like `3px` and `15px` spacing around small body text paragraphs.

We also have a theme property `space` which is an array of increasing sizes whose values are the same as the t-shirt sizes. So `theme.space[2]` is the same as `theme.verticalXs`.  This is used by `grid-styled` which we use for layout. The decision to adopt `grid-styled` was taken with Coko but so far I think we are the only project using it. I would personally favour `theme.space.xs`, as I find it easier to remember what `xs` means than a number of a scale, but this is not currently supported by `grid-styled`.

In summary I suggest we stick to `gridUnit` and `space` theme variables in the code and leave out the t-shirt sizes.

**Comment by jsms90** [comment]
@hdrury1 the 3 of us have discussed this at some length in person, but @dryhten do you want to weigh in via this issue before I make a PR with these changes?
